fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -37,14 +37,10 @@ protected void setTenant() throws Exception {
     public void testSearchQuantity_Quantity() throws Exception {
         assertSearchReturnsSavedResource("Quantity", "25|http://unitsofmeasure.org|s");
         assertSearchReturnsSavedResource("Quantity", "25||s");
+        assertSearchReturnsSavedResource("Quantity", "25");
         
-        // DSTU2 does not say if this is allowed or not, but we do not support it.
-        // In more recent versions, they clarified that it should work:  https://build.fhir.org/search.html#quantity
-//        assertSearchReturnsSavedResource("Quantity", "25");
-        
-        // I think this should return the resource but it currently doesn't.
         // https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&tracker_item_id=19597
-//        assertSearchReturnsSavedResource("Quantity", "25||sec");
+        assertSearchReturnsSavedResource("Quantity", "25||sec");
         
         assertSearchDoesntReturnSavedResource("Quantity", "24.4999||s");
```

Comment:
according to the spec https://www.hl7.org/fhir/search.html,  GET [base]/Observation?value-quantity=5.4||mg  means "Search for all the observations with a value of 5.4(+/-0.05) mg where the unit - either the code (code) or the stated human unit (unit) are "mg",   do we do the +/- value range check also?

Reply:
No, that is https://github.com/IBM/FHIR/issues/257